microsoft/testsuites/stress/tlb_stress_suite.py

```diff
@@ -109,7 +109,16 @@ def stress_tlb_basic(
                 tlb_pages=tlb_pages,
             )
         finally:
-            self._check_panic(nodes)
+            for node in nodes:
+                panic_info = node.features[SerialConsole].check_panic(
+                    saved_path=None, force_run=True
+                )
+                if panic_info:
```

Comment:
The pattern should be in the `check_panic`, and all other places should use the new kernel panic format. It doesn't need to repeat it in everywhere.

Reply:
You can still call `self._check_panic` like before, but the rest of the logic should go inside `SerialConsole.check_panic`. This pattern shows up a few times in the PR, but it’s not really necessary.